docker/exporters/Dockerfile

```diff
@@ -1,13 +1,17 @@
-FROM golang:latest as go-builder
+FROM golang:1.24 as go-builder
```

Comment:
latest always guarantee latest go compiler image. If your intend to use latest compiler I will leave it as is, otherwise it is ok to stick to concrete version. The beauty of Go is backward compatibility. All its newer compiler guarantee that old code will compile with it.

Reply:
> The beauty of Go is backward compatibility

okok, you convinced me. switching back to `latest`